### PR TITLE
Feature/obg 303 api transactions without resource

### PR DESCRIPTION
**Pull request by max402**

### CODE REVIEW

File: opba-banking-rest-api/src/main/resources/static/tpp_banking_api_ais.yml [hunk 1]
```diff
@@ -95,6 +95,46 @@ paths:
...
           $ref: './tpp_banking_api_commons.yml#/components/responses/401_UNAUTHORIZED'
 
+  /v1/banking/ais/transactions:
+    get:
+      operationId: getTransactions without account id
+      summary: Provides list of transactions by given account
+      tags:
+        - TPP Banking API Account information service (AIS)
+      description: >-
+        This operation shows how to override the global security defined above,
```
Comment (valb3r) [inline opba-banking-rest-api/src/main/resources/static/tpp_banking_api_ais.yml:105]:
```
        This operation shows how to override the global security defined above,
        as we want to open it up for all users.
```
Can't get the meaning of this phrase

File: fintech-examples/fintech-server/src/test/java/de/adorsys/opba/fintech/server/feignmocks/TppAisClientFeignMock.java [hunk 2]
```diff
@@ -51,6 +53,29 @@
...
+            String fintechRedirectURLNOK,
+            UUID xRequestID,
+            String xTimestampUTC,
+            String xOperationType,
+            String xRequestSignature,
+            String fintechID,
+            String bankID,
+            String psUConsentSession,
+            UUID serviceSessionID,
+            @Valid LocalDate dateFrom,
```
Comment (valb3r) [inline fintech-examples/fintech-server/src/test/java/de/adorsys/opba/fintech/server/feignmocks/TppAisClientFeignMock.java:70]: `@Valid` is meaningless here and down below. There is no validation present

File: opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java [hunk 3]
```diff
@@ -37,12 +39,14 @@
      * Transaction list date from.
      */
     @NotNull(message = "{no.ctx.dateFrom}")
+    @ValidationInfo(ui = @FrontendCode(STRING), ctx = @ContextCode(DATE_FROM))
```
Comment (valb3r) [inline opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java:42]: Please remove this or make PROHIBITED

File: opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java [hunk 4]
```diff
@@ -37,12 +39,14 @@
...
     private LocalDate dateFrom;
 
     /**
      * Transaction list date to.
      */
     @NotNull(message = "{no.ctx.dateTo}")
+    @ValidationInfo(ui = @FrontendCode(STRING), ctx = @ContextCode(DATE_TO))
```
Comment (valb3r) [inline opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java:49]: Please remove this or make PROHIBITED

Comment (max402) [reply opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java]: removed

File: opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java (hunk 3 above)
Comment (max402) [reply opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/Xs2aTransactionParameters.java]: removed

**Pull request merged**